RELEASE-README.md

```diff
@@ -414,86 +402,78 @@ If everything is perfect (compiles, Jenkins is all blue, sanity checks succeed a
+        $ ./droolsjbpm-build-bootstrap/script/release/update-version-all.sh 7.x.x-SNAPSHOT community
         ```
 
         * Commit those changes:
 
             ```shell
-            $ droolsjbpm-build-bootstrap/script/git-all.sh add .
+            $ ./droolsjbpm-build-bootstrap/script/git-all.sh add .
 
-            $ droolsjbpm-build-bootstrap/script/git-all.sh commit -m "Set next development version: 6.3.0-SNAPSHOT"
+            $ ./droolsjbpm-build-bootstrap/script/git-all.sh commit -m "Set next development version: 7.x.x-SNAPSHOT"
             ```
 
         * Push all changes to the blessed repository:
 
             ```shell
-            $ droolsjbpm-build-bootstrap/script/git-all.sh push
-            ```
-
-        * Warning: If releasing from master (i.e. a Beta release) and the push fails as there have been other commits to the remote master branch it might be necessary to pull.
-
-            ```shell
-            $ droolsjbpm-build-bootstrap/script/git-all.sh pull
+            $ ./droolsjbpm-build-bootstrap/script/git-all.sh push origin master
             ```
+    
+* Release your staging repository on [nexus](https://repository.jboss.org/nexus) and push the tags to GitHub
 
-    * Checkout back to your local release branch.
+    * Nexus: Button *release*
+    
+    * locally: Checkout back to your local release branch.
 
         ```shell
-        $ git-all.sh checkout r6.2.0.Final
+        $ ./droolsjbpm-build-bootstrap/script/git-all.sh checkout r7.x.x.Final
         ```
 
-
-* Push the local tag from the local release branch to the remote blessed repository.
-
-      $ droolsjbpm-build-bootstrap/script/release/git-push-tag-all.sh 6.2.0.Final 6.2.0.Final
-
-    * Push your changes to the release branch:
-
-        * Especially if the release branch is master: First pull any latest changes **without `--rebase`**, .
+        * Create the tag locally:
 
             ```shell
-            $ git-all.sh pull
+            $ ./droolsjbpm-build-bootstrap/script/release/git-all.sh -a 7.x.x.Final -m ·tagged 7.x.x.Final"
             ```
 
-            * Without the `--rebase` it's a merge, and their commits will not be rebased before your version-changing commits.
-
-        * Push your version-changing commits to the release branch:
+        * Push the local tag from the local release branch to the remote blessed repository.
 
             ```shell
-            $ git-all.sh push origin 5.2.x
-            ```
-
-* Release your staging repository on [Nexus](https://repository.jboss.org/nexus)
-
-    * Button *release*
+            $ ./droolsjbpm-build-bootstrap/script/git-all.sh push <tag> | where <tag> is 7.x.x.Final    
 
-* Go to [JIRA](https://issues.jboss.org) and for each of our JIRA projects (DROOLS, PLANNER, JBPM, GUVNOR):
+* Go to [JIRA](https://issues.jboss.org) and for each of our JIRA projects (DROOLS, OPTAPLANNER, JBPM, APPFORMER):
```

Comment:
The jira URL is now https://issues.redhat.com, even issues.jboss.org works, it is just due HTTP redirection.
The list of jira project was correct:

- DROOLS -> https://issues.redhat.com/browse/DROOLS,
- PLANNER - https://issues.redhat.com/browse/PLANNER,
- JBPM -> https://issues.redhat.com/browse/JBPM,
- Appformer -> https://issues.redhat.com/browse/AF